### DISCUSSION TITLE
Publications on V Programming Language?

### DISCUSSION THREAD

**Discussion by kevinmatthes** [vlang/v#18766] [Questions and Answers] [answered]
I would like to learn V and was wondering whether there might be some recommended introductory books and / or papers on the language in a addition to https://github.com/vlang/v/blob/master/doc/docs.md as it is indeed interesting how different authors focus on certain aspects of a language such that each work still adds something new.

**Comment by JalonSolov** [comment, marked as answer]
From the Draft wikipedia page (they still haven't approved it to go live...):

["The V Programming Language basic" (in Japanese). June 20, 2020](https://www.amazon.com/dp/B08BKJDRFR)

["Getting Started with V Programming" - Rao, Navule Pavan Kumar, December 10, 2021](https://www.worldcat.org/oclc/1290492862)

["Beginning with V Programming" - Lyons, Dakota "Kai", April 13, 2022](https://en.wikipedia.org/wiki/Special:BookSources/979-8801499963)

"[Discover the V language" -Tsoukalos, Mihalis, May 2022 Linux Format Magazine #288](https://www.worldcat.org/issn/1470-4234)
